internal/testcafe/config.go

```diff
@@ -186,7 +191,7 @@ func SplitSuites(p Project) (Project, Project) {
 	var dockerSuites []Suite
 	var sauceSuites []Suite
 	for _, s := range p.Suites {
-		if s.Mode == "docker" {
+		if s.Mode == "docker" || p.Defaults.Mode == "docker" {
```

Comment:
Seems default mode setting is missing for testcafe just like cypress and playwright https://github.com/saucelabs/saucectl/blob/main/internal/cmd/run/cypress.go#L52

with above modification, I don't think it should check default mode here

Reply:
Just for posterity (since discussion happened offline): The application of `mode` defaults onto the suite existed in https://github.com/saucelabs/saucectl/pull/348/files/2ab6fb7346a9876cbd14aa4cd5b839cc450d680b#diff-84f1fbdd75faa83342c33964a2d7ca8e18a58cd5fcd7090d9af434027deff641L134, but was removed in favor of doing the check when it was actually needed, i.e. at the time of the split betweend docker & sauce.